app/src/test/kotlin/com/wire/android/ui/home/conversations/ConversationsViewModelTest.kt

```diff
@@ -61,6 +70,7 @@ class ConversationsViewModelTest {
 
         // Default empty values
         coEvery { getMessages(any()) } returns flowOf(listOf())
+        coEvery { observeMemberDetails(any()) } returns flowOf(listOf())
```

Comment:
Why do we need to declare these initially if we are overriding them on each test upon our needs? Maybe check the Arrangement pattern I used in `SendImageUseCaseTest`. It could help to reduce a lot of the boilerplate code as well as organising these logical tests a bit better.

Reply:
Some of the other tests here don't care at all about these flows, so I just initialized them as empty in order to keep the other tests working.

I'll consider this pattern next time 👍🏼